packages/blade/src/utils/useVerifyAllowedChildren/useVerifyAllowedChildren.ts

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { getComponentId } from '../isValidAllowedChildren';
+import { throwBladeError } from '../logger';
+
+/**
+ * Verify if the passed childrens are only of allowedComponents list
+ */
+const useVerifyAllowedChildren = (props: {
```

Comment:
how is this diff than isValidAllowedChildren?

Reply:
`isValidAllowedChildren` only does this part `getComponentId(component) === id` nothing else. 

We need a more reusable hook which loops over React childrens and validates based on given `allowedComponents` array.